data/data-pipeline/data_pipeline/tests/sources/example/test_etl.py

```diff
@@ -17,19 +16,30 @@
 
 class TestETL:
     """A base class that can be inherited by all other ETL tests.
-
     Note: every method that does *not* need to be reimplemented by child classes has
     the test name pattern of `test_*_base`. All other tests need to be reimplemented.
+    This uses pytest-snapshot.
+
+    To update individual snapshots: $ poetry run pytest
+            data_pipeline/tests/sources/national_risk_index/test_etl.py::TestClassNameETL::<testname>
+            --snapshot-update
     """
 
     # In every child test class, change this to the class of the ETL being tested.
     _ETL_CLASS = ExampleETL
 
     # The following constants do not need to be updated in child class.
-    _INPUT_CSV_FILE_NAME = "input.csv"
+    _EXTRACT_CSV_FILE_NAME = "extract.csv"
```

Comment:
I've left a snapshot for this in right now (@esfoobar-usds) but renamed it so it's less confusing.